### PR TITLE
Changes Frontier Alliance naming, Removes ATLAS

### PR DESCRIPTION
**Pull request by LadyFowls**
- tweak: "Refactors Frontier Alliance to Coalition namesake due to lore changes."

### CODE REVIEW

**Comment by LadyFowls** [comment]
!review

File: code/modules/background/citizenship/human.dm [hunk 1]
```diff
@@ -78,9 +78,9 @@
 	and poor, yet free-willed and provocative Dregs. The Federation is controlled by several megacorporations, and although NanoTrasen is not one of them, it is a common former \
 	workplace of various contractors on board NSS Aurora."
 
-/datum/citizenship/frontier
-	name = CITIZENSHIP_FRONTIER
-	description = "Rising from the former Coalition of Colonies, the Frontier Alliance is a loose confederation of united \"entities\" within the human frontier. These entities range \
+/datum/citizenship/coalition
+	name = CITIZENSHIP_COALITION
+	description = "The Coalition of Colonies is a loose confederation of united \"entities\" within the human coalition. These entities range \
```
Comment (alsoandanswer) [inline code/modules/background/citizenship/human.dm]:
```suggestion
	description = "The Coalition of Colonies is a loose confederation of united \"entities\" within the human frontier. These entities range \
```
>coalition is a loose confederation within the human coalition
epic

**Comment by Alberyk** [comment]
Remove the custom item sprites too.

**Comment by skull132** [comment]
Approved by pegageese.

**Pull request merged**